### PR TITLE
More info for the website

### PR DESCRIPTION
**Pull request by iamjohnbarker**
This is an addition to the initial website with the inclusion of the
OSC API options. Mostly copy/pasted from the github with lots of little
changes for consistency and ease of reading.

### CODE REVIEW

File: website/index.html [hunk 1]
```diff
@@ -41,19 +44,301 @@ <h2>OSC-bridge for controlling ATEM switchers</h2>
...
+            </ul>
+            <p>
+              Feedback: None.
+            </p>
+            <h3>Auxiliary source selection</h3>
+            <ul>
+              <li>
+                Aux <span class="callout">/atem/aux/$i $x</span>
+                <ul>
+                  <li>Where <span class="callout">$x</span> is an integer value that is a valid program source, and can be 1-6 depending on the capability of your ATEM switcher. Check the Help Menu for the correct values.</li>
```
Comment (sneat) [inline website/index.html]: This is probably an issue in the actual codebase as well, but it should be:

`<li>Where <span class="callout">$x</span> is an integer value that is a valid AUX output, and can be 1-6 depending on the capability of your ATEM switcher. Check the Help Menu for the correct values.</li>`

**Comment by iamjohnbarker** [comment]
@sneat - Good catch! As for the initial use case, I was going to go down the OSCulator route, but maybe you have a suggestion of a similar "getting started" use case that gets right to the point. Is OSCulator a little too advanced or is that the best place to start?

**Comment by danielbuechele** [comment]
From what I hear many people using atemOSC together with Osculator, so I think this would be a good use-case to have in the docs. However, perhaps this shouldn't be the only usecase we list. I don't know which other OSC-clients are usually used. QLab?

**Comment by sneat** [comment]
I think we should have decent tutorials/getting started type things for OSCulator, TouchOSC and QLab. I can certainly help with the QLab one at some point.

**Comment by iamjohnbarker** [comment]
@sneat help with QLab will be great - I'll tackle the other two options (TouchOSC and OSCulator) first and then see how they go!
@danielbuechele are we good to merge this and then add the other things later? or do you want me to just hold off and do it all in one go?

**Comment by danielbuechele** [comment]
Feel free to merge and update step by step.

**Comment by SteffeyDev** [comment]
@iamjohnbarker did you see my comments above?

**Comment by iamjohnbarker** [comment]
@SteffeyDev Nope, I don't see any comments from you on this pull request.

**Comment by danielbuechele** [comment]
Check out the deployed site: http://danielbuechele.github.io/atemOSC/
Great work @iamjohnbarker

**Comment by SteffeyDev** [comment]
Hm, I started a review on the code, maybe other people can’t see it? I just mentioned that the “atemOSC.app” should not have space between “atemOSC” and “.app”, and that in the issues section, not everyone will have to create an account, so it should probably be “create an account or log in (if needed)”

**Comment by SteffeyDev** [comment]
@iamjohnbarker thanks for the updates though!

**Comment by iamjohnbarker** [comment]
@SteffeyDev Ahh crap - sorry I missed your review. I'm not even sure where that would show up - could you point me in the right direction?
More than happy to include those changes in an upcoming release!

**Pull request merged**